src/command_modules/azure-cli-sf/HISTORY.rst

```diff
@@ -1,10 +1,12 @@
 .. :changelog:
 
-Release History
+Unreleased History
 ===============
 
+1.0.9 (2017-09-14)
+* Revert 'Deprecating all commands in favor of Service Fabric CLI (sfctl)'
```

Comment:
That's not what I meant.
More like this...

```
Release History
===============

unreleased
++++++++++
* Revert 'Deprecating all commands in favor of Service Fabric CLI (sfctl)'
```

see https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log